root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
-          </a>
-        ) : null}
-      </p>
-
-      <p className="right">
-        {exp.l(
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="bs footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3 className="ms-3">
+              <img
+                alt="MusicBrainz"
+                src={logo}
+                width="196"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <ul className="list-unstyled ms-3">
+              <li>
+                <span className="fs-5">
+                  {l('Development IRC:')}
+                </span>
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Discussion IRC:')}
                 </span>
-              ),
-            })}
-          </>
-        ) : null}
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#musicbrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#musicbrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Email:')}
+                </span>
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="mailto:support@metabrainz.org"
+                >
+                  {l('support@metabrainz.org')}
+                </a>
+              </li>
```

Comment:
Make sure Rob is ok with this. Right now we do not show the email address on https://metabrainz.org/contact until the user clicks an acknowledgement that they won't contact us regarding a bunch of bullshit questions we used to get a lot more often than we do now - I'm a bit afraid it will get worse again by doing this. I might consider just linking to https://metabrainz.org/contact instead.

Reply:
Right